package.json

```diff
@@ -60,7 +60,8 @@
     "compression": "^1.4.4",
     "configstore": "^3.0.0",
     "console-ui": "^1.0.2",
-    "core-object": "^3.0.0",
+    "core-object": "^3.1.3",
+    "dag-map": "^2.0.2",
```

Comment:
and it was a missing dependency previously? or why is this being added here? seems unrelated to the Node 8 update